src/datumaro/plugins/data_formats/coco/page_mapper.py

```diff
@@ -5,7 +5,10 @@
 import logging as log
 from typing import Any, Dict, Iterator, List, Optional, Tuple
 
-from datumaro.rust_api import CocoPageMapper as CocoPageMapperImpl
+try:
+    from datumaro.rust_api import CocoPageMapper as CocoPageMapperImpl
+except ImportError:
+    pass
```

Comment:
What is the exact case requiring this fix? If any interpreter goes into L10-11, it means that it doesn't loads `CocoPageMapperImpl` just a once but does not load `CocoPageMapperImpl` totally.

Reply:
The error occurs when importing a dataset multiple times in the streamlit GUI. 
I thought this problem can be reproduced easily in the normal interpreter, but it was not.
I guess I need to handle this problem in another way.
So, I'll cancel this change at the moment.

Reply:
What Python version is in your dev env? 3.12?